### ISSUE TITLE
How to set Namespace in proxy generator.

### ISSUE THREAD

**Issue by DonaldRecord** [jweber/WcfClientProxyGenerator#27]
Hello,

Is there any way in WCF Client Proxy Generator to programmatically set the binding namespace of the created service proxy?

**Comment by jweber** [comment]
I don't believe there is currently a way to accomplish this. If you can point me to any documentation or examples of this being done elsewhere I can take a look at implementing it into the library.

**Comment by DonaldRecord** [comment]
Sorry for the delayed response. There are several Git projects for this same purpose, but none of them seem to have this functionality. To be entirely honest, I am not sure that .NET allowed a way for it to be done dynamically. I will do more research myself when I have time. Here are some links to start.

[https://stackoverflow.com/questions/7364024/service-contract-namespace-why](url)
[https://msdn.microsoft.com/en-us/library/system.servicemodel.servicecontractattribute(v=vs.110).aspx](url)
[(https://social.msdn.microsoft.com/Forums/vstudio/en-US/07b30bd4-8a12-4447-9e80-30d2bad85abe/wcf-service-contract-namespace?forum=wcf)](url)
[https://docs.microsoft.com/en-us/dotnet/framework/wcf/service-versioning](url)
